src/common/components/repository-row/dropdowns/configuration-error-dropdown.js

```diff
@@ -0,0 +1,61 @@
+import React, { PropTypes } from 'react';
+
+import { Row, Data, Dropdown } from '../../vanilla/table-interactive';
+
+import getTemplateUrl from './template-url.js';
+
+const ConfigurationErrorDropdown = ({ snap }) => {
+
+  // XXX
+  // to access info about yaml parse error (YAMLException, from js-yaml module)
+  //
+  //  snap.snapcraftData.error = {
```

Comment:
A comment with details how to access yaml parsing error from the component if we want to show detailed validation error message.